### DISCUSSION TITLE
Projects (beta): Excluding a field value in a grouped view should hide that group

### DISCUSSION THREAD

**Discussion by JasonTheAdams** [community/community#5224] [Projects and Issues]
We have a a view that groups by the status. In this view, we want to only show statuses that are active (new, in progress, and so forth). Completed items should not show up, so they were excluded.

I found, though, that the "Completed" group is still visible, though, it's just always empty. This makes for unnecessary visual clutter. The group should be hidden entirely if it's excluded.

**Comment by archon810** [comment]
We're just starting with iterations (sprints) too, and right now have 3, but when we get to 23, I assume the view that groups by iterations will get very wide, and the option to hide empty iterations following an applied filter (such as `sprint:@current`) would be very much necessary.

**Reply by Aaron-Comsense** [reply]
We have a view that has iterations as the columns, and the view automatically hides iteration columns that are more than 4 iterations old. One would think this is great as the view isn't forever wide. However, all issues that are in an iteration older than 4 iterations are displayed in the "No Iteration" even though the issue is assigned to an iteration.

![image](https://user-images.githubusercontent.com/71103707/180069414-74d5b3a1-6837-491f-8c7e-f6bc4f4c235e.png)
